### PR TITLE
chore(docs): fix typos in migrating-from-v1-to-v2

### PR DESCRIPTION
**Pull request by RahulMahesh62**
## Description

Some typos were found in migrating-from-v1-to-v2.md and they were fixed. This is an internal doc.

## Related PR

This PR links to #26189

### CODE REVIEW

File: docs/docs/migrating-from-v1-to-v2.md [hunk 1]
```diff
@@ -441,7 +441,7 @@ const Photo = ({ location, photoId }) => {
 
 React Router would pass a `history` prop to components that you could use to navigate.
 
-If you need to do programmatic navigation, import instead the @reach/router's `navigate` function.
+If you need to do programmatic navigation, import @reach/router's navigate function instead.
```
Comment (LekoArts) [inline docs/docs/migrating-from-v1-to-v2.md:444]:
```suggestion
If you need to do programmatic navigation, import @reach/router's `navigate` function instead.
```

Please keep it in code blocks

File: docs/docs/migrating-from-v1-to-v2.md [hunk 2]
```diff
@@ -945,7 +945,7 @@ See an example in [this PR that upgrades the `using-remark` site to Gatsby v2](h
 
 If your Gatsby v1 site included any polyfills, you can remove them. Gatsby v2 ships with Babel 7 and is configured to automatically include polyfills for your code. See [Gatsby's Babel docs for more details](/docs/babel).
 
-> Note: This works for your own code, but is not yet implemented for code imported from `node_modules`. Track progress of this feature in [this issue](https://github.com/gatsbyjs/gatsby/issues/7064).
+> Note: This works for your code, but is not yet implemented for code imported from `node_modules`. Track the progress of this feature in [this issue](https://github.com/gatsbyjs/gatsby/issues/7064).
```
Comment (LekoArts) [inline docs/docs/migrating-from-v1-to-v2.md:948]: You can remove this note as it has been implemented

**Review by LekoArts — CHANGES_REQUESTED** [review]
Two small comments, thanks for the PR!

File: docs/docs/migrating-from-v1-to-v2.md (hunk 2 above)
Comment (RahulMahesh62) [reply docs/docs/migrating-from-v1-to-v2.md:948]: So, should I remove the present note ?

Comment (LekoArts) [reply docs/docs/migrating-from-v1-to-v2.md]: Yes! :)

**Comment by RahulMahesh62** [comment]
As of now, codeblocks typo has been done. Also the note was removed

**Review by LekoArts — APPROVED** [review]
Thanks! 👍

**Comment by RahulMahesh62** [comment]
Happy to know that I contributed to Gatsby..🤩🎉

**Pull request merged**